janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIoTest.java

```diff
@@ -93,15 +93,15 @@ public void setup() {
     @ParameterizedTest
     @MethodSource("data")
     public void testSerialization(String name, Function<Graph, GraphReader> readerMaker, Function<Graph, GraphWriter> writerMaker) throws Exception {
-        testSerializationSpecialized(null, readerMaker, writerMaker);
-        testSerializationSpecialized(makeLine,readerMaker, writerMaker);
-        testSerializationSpecialized(makePoly,readerMaker, writerMaker);
-        testSerializationSpecialized(makeMultiPoint,readerMaker, writerMaker);
-        testSerializationSpecialized(makeMultiLine,readerMaker, writerMaker);
-        testSerializationSpecialized(makeMultiPolygon,readerMaker, writerMaker);
+        testSerializationSpecialized("", null, readerMaker, writerMaker);
```

Comment:
Nit. In the event this ever triggered a failure a name specifying it's a null input, like `nullGeoshape` or `nullInput` would be preferable to an empty string.